model/project_ref.go

```diff
@@ -142,6 +143,15 @@ type TriggerDefinition struct {
 	Alias        string `bson:"alias,omitempty" json:"alias,omitempty"`
 }
 
+type PatchTriggerDefinition struct {
+	DefinitionAlias   string `bson:"definition_alias" json:"definition_alias"`
```

Comment:
is alias sufficient to be used here? definition doesn't seem to add anything

Reply:
done